### DISCUSSION TITLE
Creating the project

### DISCUSSION THREAD

**Discussion by AkashWasTaken** [Minestom/Minestom#2383] [Help]
Hi, I am a really dumb guy whos new to java, anyone can help me setup a minestom server with gradle in vscode (just tell me the instruction, im confused with gradle init)

**Comment by xMrAfonso** [comment]
Not the help you wanted but I think it's the one you need:
- If you have no knowledge with java, I would suggest learning the basics of Java before even touching Minestom.
- I would then learn Gradle Basics.
- I would then install an actual Java IDE (like Intellij) since VSCode simply sucks for java.
- I would then either watch a course or just look around in the javadocs and try to create simple stuff for training.

**Reply by AkashWasTaken** [reply]
thanks I learned java basics and because intellij IDE is resource intense it lags alot in my laptop so that's the reason Im using vscode.

**Reply by mzcydev** [reply]
Try Eclipse.

**Reply by jordqubbe** [reply]
Yeah if you're going to code in Java, use a dedicated Java IDE. Try eclipse first if you can't use IntelliJ, and then Netbeans if eclipse doesn't work. If you're looking for a "tutorial", find Kody Simpson on YouTube, he makes good videos about Minestom.

**Reply by AkashWasTaken** [reply]
I have fixed the issue by using maven instead thanks anyways
